domain/src/commonMain/kotlin/com/islamversity/domain/AyaOption.kt

```diff
@@ -0,0 +1,11 @@
+package com.islamversity.domain
+
+inline class AyaOption(val enable : Boolean){
```

Comment:
an enum would make more sense here I guess
but if you want to stay away from the type safety net
then the boolean should work too